core/src/main/scala/kafka/log/LogLoader.scala

```diff
@@ -172,7 +173,7 @@ object LogLoader extends Logging {
   private def removeTempFilesAndCollectSwapFiles(params: LoadLogParams): Set[File] = {
 
     def deleteIndicesIfExist(baseFile: File, suffix: String = ""): Unit = {
-      info(s"${params.logIdentifier} Deleting index files with suffix $suffix for baseFile $baseFile")
+      info(s"${params.logIdentifier}Deleting index files with suffix $suffix for baseFile $baseFile")
```

Comment:
is the change (remove blank space here) along with https://github.com/apache/kafka/pull/10742/files#diff-54b3df71b1e0697a211d23a9018a91aef773fca0b9cbd1abafbdca6c79664930L70-R70 (add blank space there) needed? [I am not sure the best practice in Kafka codebase]

Reply:
Yes, because the existing identifier in the `Log` class is suffixed with a whitespace, see: https://github.com/apache/kafka/blob/5d4f9f917c8dc356a2d922980ba8101e0f2e7093/core/src/main/scala/kafka/log/Log.scala#L280.